docs/admin/network-plugins.md

```diff
@@ -39,7 +39,7 @@ The Linux-only kubenet plugin provides functionality similar to the `--configure
 
 The plugin requires a few things:
 
-* The standard CNI `bridge` and `host-local` plugins to be placed in `/opt/cni/bin`.
+* The standard CNI `bridge` and `host-local` plugins to be placed in `/opt/cni/bin`. Or specify `network-plugin-dir` to supply additional search path for CNI plugins.  
```

Comment:
then drop the or and say /opt/cni always takes precedence. if we find it in /opt/cni do we still look in network-plugind-dir? (worth mentioning)